### PR TITLE
Describe behavior of createChar when supplying a name and the character exists

### LINKED ISSUE #69: Python createChar() documentation should be updated
http://fontforge.10959.n7.nabble.com/Python-createChar-misbehaves-if-a-glyph-already-exists-at-the-given-unicode-codepoint-td7280.html

### PR DESCRIPTION
**Pull request by skef**
I guess I'm making docs changes now after all.

Closes #69

**Note: Not a sexual reference**

### CODE REVIEW

**Comment by ctrlcctrlv** [comment]
@jtanx Are you OK with docs changed being merged?

**Comment by jtanx** [comment]
As long as I'm aware of it, so I can keep my working copy up to date, sure

File: doc/html/python.html [hunk 1]
```diff
@@ -4512,8 +4512,11 @@ <H2 id="Font">font</H2>
       <TD><CODE>(uni[,name])</CODE></TD>
       <TD>Create (and return) a character at the specified unicode codepoint in
 	this font and optionally name it. If you wish to create an glyph with no
```
Comment (jtanx) [inline doc/html/python.html]: unrelated typo here `an glyph` -> `a glyph`

Comment (skef) [reply doc/html/python.html]: Updated

**Comment by fitojb** [comment]
> Note: Not a sexual reference

Stop trying to be funny, you’re not going to do stand-up any time soon.

**Comment by ctrlcctrlv** [comment]
@fitojb, your comment is completely unnecessary. If you're so busy as your profile claims, don't waste my time having to read your nonsense. You've engaged in trolling here before, (in my humble opinion,) so let this serve as a warning.

@skef The joke was very funny and I'm expecting at least one joke per PR now.

**Comment by skef** [comment]
@fitojb Sometimes a small amount of humor is necessary to stave off further attempts at humor. The minimization of fun is a delicate art and I am only an amateur -- have some sympathy and patience for one with less talent for it than yourself.

**Pull request merged**